modules/document_repository/README.md

```diff
@@ -0,0 +1,35 @@
+# Document Repository
+
+## Purpose
+
+ The Document Repository is a tool that provides studies with a centralized location for non-candidate documents. Files may be uploaded and organized under any user-defined category.
+
+## Intended Users
+ 
+ All Loris user and data entry staff could view, update, edit and delete the upload files with right permissions.
+
+## Scope
+
+ It's for all non-candidate documents.
```

Comment:
Change: "non-candidate documents" to "documents relevant to the study across multiple candidates  (as opposed to Media module)"